todo-plainJS.js

```diff
@@ -0,0 +1,154 @@
+let taskList = JSON.parse(localStorage.getItem('taskList')) ?? [];
+
+window.addEventListener('DOMContentLoaded', () => {
+    drawHTML(filterTasks());
+    refreshCount();
+});
+
+form.addEventListener('submit', (event) => {
+    event.preventDefault();
+    const content = task_content.value;
+
+    if (content === '') {
+        console.log('Input is empty');
+    } else {
+        createTask(content);
+        createTaskHTML(content);
+    }
+
+    saveTaskList();
+    refreshCount();
+    task_content.value = '';
+});
+
+for (let item of filters.children) {
+    item.addEventListener('click', () => {
+        switch (item.id) {
+            case 'filter_done':
+                drawHTML(filterTasks('done'));
+                refreshCount(filterTasks('done'));
+                break;
+            case 'filter_undone':
+                drawHTML(filterTasks('undone'));
+                refreshCount();
+                break;
+            default:
+                drawHTML(filterTasks());
+                refreshCount();
+                break;
+        }
+    });
+}
+
+todo.addEventListener('click', (event) => {
+    const index = Array.from(todo.children).indexOf(event.target);
+
+    if (event.target.parentNode === todo) {
+        event.target.classList.toggle('done');
+        console.log('test');
+        markTask(index);
+        saveTaskList();
+        refreshCount();
+    }
+});
+
+todo.addEventListener('click', (event) => {
+    const index = Array.from(todo.children).indexOf(event.target.parentNode);
+    
+    if (event.target.classList.contains('task-delete_button')){
+        event.target.parentElement.remove();
+        deleteTask(index);
+        saveTaskList();
+        refreshCount();
+    }
+});
+
+function drawHTML(taskList) {
+    while (todo.firstChild) {
+        todo.removeChild(todo.firstChild);
+    }
+
+    taskList.forEach((item) => {
+        createTaskHTML(item.content, item.isDone);
+    });
+}
+
+function createTaskHTML(content, isDone = false) {
+    const task = document.createElement('li');
+    const classes = task.classList;
+    task.textContent = content;
+    classes.add('task');
+
+    if (isDone) {
+        classes.add('done');
+    }
+
+    todo.append(task);
+
+    const doneIcon = document.createElement('span');
+    doneIcon.classList.add('task-done_icon');
+    task.prepend(doneIcon);
+
+    const deleteButton = document.createElement('span');
```

Comment:
You have two buttons here: task with circle itself and delete button, but they are "li" and "span". Consider to change them to the button. (Put button inside li as a task and delete button as sibling of this one)